packages/@aws-cdk/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -92,6 +92,13 @@ export interface ApplicationLoadBalancedFargateServiceProps extends ApplicationL
    * @default - A new security group is created.
    */
   readonly securityGroups?: ISecurityGroup[];
+
+  /**
+   * The runtime platform of the task definition
+   *
+   * @default undefined If not specified, `operatingSystemFamily` is LINUX and `cpuArchitecture` is X86_64
```

Comment:
Do not use the value undefined in describing @default values. This is part of the language specification. Instead, use a dash (-) as the first word and describe the behavior of the infrastructure the construct exhibits when the value is not supplied (See [Contributing Guide, Defaults](https://github.com/aws/aws-cdk/blob/v1-main/docs/DESIGN_GUIDELINES.md#defaults)).